src/jshint.js

```diff
@@ -3413,88 +3409,31 @@ var JSHINT = (function() {
     });
   }
 
-  var conststatement = stmt("const", function(context) {
+  function variableStatement(type, statement, context) {
```

Comment:
I'm used to using `debugger;` statements, so the annoyance of conditional breakpoints doesn't bother me too much. I just wish there was a better name than "variable", since it heavily suggests `var` and is somewhat contradictory with "constant". A function like `lexicalDeclaration` would avoid this, but I'm with @lukeapage w/r/t drift. Let's run with this approach for now.